README.md

```diff
@@ -1,2 +1,196 @@
-# helm-operator
-Helm Operator for OpenEBS Installation
+# OpenEBS Helm Operator
+
+## Introduction
+
+- This is  a helm operator for openebs
+- The content of values.yaml will be placed as spec items in `OpenEBSInstallTemplate` CR 
+- The operator watches all namespaces. There is no need for helm client/tillerto be installed.
+  (the helm operator imports code from helm project)
+- Changes to spec of the CR will behave similar to values override via helm
+- All OpenEBS resources have ownerReference of `OpenEBSInstallTemplate` & are deleted upon removal of CR
+
+Example of OpenEBSInstallTemplate: 
+
+  ```yaml
+  apiVersion: openebs.io/v1alpha1
+kind: OpenEBSInstallTemplate
```

Comment:
How is this name determined / understood by Helm?
Can we name this to `OpenEBSTemplateValues`?
I am confused if this is Template or Template Values?

Reply:
This value need not be understood by helm @AmitKumarDas . The helm operator merely looks at the `spec.` items and interprets as values.yaml - to override what is present in the bundled/embedded helm chart. 

The reason we have `Install` in the CR name is to ensure it is understood by users as an "Install" operator as against a traditional lifecycle manager.